scripts/cc-cli/src/commands/setKeys.ts

```diff
@@ -0,0 +1,36 @@
+import { Command, OptionValues } from "commander";
+import { newApi } from "../api";
+import { getSeedFromOptions, initKeyringPair } from "../utils/account";
+
+export function makeSetKeysCommand() {
```

Comment:
Not necessary for this PR, but as a user it might be nice to have a rotate + set keys command instead of having to call them in sequence